tests/js/karma.conf.js

```diff
@@ -91,6 +91,6 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: true
+    singleRun: false
```

Comment:
@smarnach CI build was changed to pass `--single-run` command line parameter (takes precedence over conf file) in [this commit](https://github.com/open-craft/xblock-group-project-v2/commit/16553537b3a4aa36be062dae1103482958bb94f7). Basically, this change should be the part of that commit, but I missed it than.